packages/terra-status-view/src/StatusView.jsx

```diff
@@ -0,0 +1,210 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import Divider from 'terra-divider';
+import styles from './StatusView.scss';
+
+const cx = classNames.bind(styles);
+
+const variants = {
+  NODATA: 'no-data',
+  NOMATCHINGRESULTS: 'no-matching-results',
+  NOTAUTHORIZED: 'not-authorized',
+  ERROR: 'error',
+  CUSTOM: 'custom',
+};
+
+const propTypes = {
+  /**
+   * An array of actionable buttons to display.
+   */
+  children: PropTypes.node,
+
+  /**
+   *  Determines if the content should be aligned vertically at the top of the container rather than in the middle.
+   */
+  isAlignedTop: PropTypes.bool,
+
+  /**
+   * Determines if the glyph should be displayed.
+   */
+  isGlyphHidden: PropTypes.bool,
+
+  /**
+   * The descriptive text to display.
+   */
+  message: PropTypes.string,
+
+  /**
+   * The required status title to display. Status views with variants of type no-data, no-matching-results,
+   * not-authorized, and error will have defaulted values unless overridden with this prop.
+   */
+  title: PropTypes.string,
+
+  /**
+   * Determines the type of icon to display.
```

Comment:
List the variants in the description so people know what they options are. 

Also nit: I would remove the use of 'type of icon' here given other components have both types and variants and it might be confusing to consumers.

Reply:
listed the variants,

changed to use the term glyph